add-on/panels/css/desktop.css

```diff
@@ -0,0 +1,215 @@
+/* Room share panel */
```

Comment:
Please add a license version block to the top of this file - you can use the same ones as for desktopViews.jsx